Yosemite/Yosemite/Actions/JetpackConnectionAction.swift

```diff
@@ -1,8 +1,11 @@
 import Foundation
+import Networking
 import WordPressKit
 
 /// Defines actions supported by `JetpackConnectionStore`.
 public enum JetpackConnectionAction: Action {
     /// Fetches the URL used for setting up Jetpack connection using the given authenticator
-    case fetchJetpackConnectionURL(siteURL: String, authenticator: Authenticator, completion: (Result<URL, Error>) -> Void)
+    case fetchJetpackConnectionURL(completion: (Result<URL, Error>) -> Void)
+    /// Fetches the user connection state with the given site's Jetpack and authenticator.
```

Comment:
super nit: does this action still need the authenticator as mentioned in the comment?